packages/design/src/animation.mdx

```diff
@@ -0,0 +1,47 @@
+---
+name: Animation
+menu: Design
+route: /animation
+---
+
+# Animation
+### Patterns
+
+Animations in Jobber fall under categories based on their purpose:
+
+**Reveal** 
+
+Reveals are used to show and hide information on the screen. For example, toasts, modals, sidebars appearing and disappearing on the screen.
+
+**State change**
+
+State changes are used to inform the user that a context change is taking place. This is used when the object has changed state due to user interaction (e.g. hover and focus states).
+
+**Emphasis**
+
+Emphasis is used to bring attention to something on already on the screen. For example, when you want to guide users to a particular part of the interface for onboarding or learning purposes.
+
+**Delighter**
+
+Delighters can be used occasionally to create surprise and delight, and can give personality to our brand. These can be used to reinforce success in an important milestones in a user's journey.
+
+### Timing
+Timing is used for animation durations.
+
+Use judgement when determining the appropriate timing. Duration will depend on the size of the element and the distance it covers. Animations for larger elements or longer distances should be longer than smaller elements or shorter distances.
+
+
+| Name        | Value                  | Examples of use              |
```

Comment:
I really like the examples of use here

Reply:
If we are calling these "Timing" options, can we change the variables to `timing` ie `--timing-fast`?

Reply:
Makes sense we use one or the other. Framer motion uses "duration" so I've renamed this section